routes/panel.js

```diff
@@ -9,189 +9,194 @@ var shows = require('../database/shows');
 var faqs = require('../database/faqs');
 var messages = require('../database/messages');
 
+// Image compression module
+var Jimp = require('jimp');
```

Comment:
It'd be nice if this was also `const`. Actually, const everywhere for variables that aren't modified are 👍